ihmc-communication/src/main/java/us/ihmc/communication/ros2log/RecordTopicManager.java

```diff
@@ -0,0 +1,72 @@
+package us.ihmc.communication.ros2log;
+
+import gnu.trove.list.array.TLongArrayList;
+import org.apache.commons.lang3.tuple.Pair;
+import us.ihmc.ros2.ROS2Node;
+import us.ihmc.ros2.ROS2Topic;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.LongSupplier;
+
+class RecordTopicManager<T>
```

Comment:
I would prefix these helper classes with ROS2Log to show they're part of that framework

Reply:
For package-private classes I use the convention that name prefixes aren't necessary.